### PR TITLE
Avoid unnecessary panic by ignoring nil values in maps

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

File: uast/node.go [hunk 1]
```diff
@@ -305,6 +306,8 @@ func (c *ObjectToNode) toNodes(obj interface{}) ([]*Node, error) {
 			}
 
 			n.Children = append(n.Children, children...)
+		case nil:
```
Comment (mcuadros) [inline uast/node.go]: is not covered by any test.

File: etc/build/etc/build/Dockerfile.build.alpine.tpl [hunk 2]
```diff
@@ -11,20 +11,28 @@ RUN set -ex \
...
-	&& rm golang.tar.gz \
-	&& cd /usr/local/go/src \
-	&& patch -p2 -i /no-pic.patch \
-	&& ./make.bash \
-	\
-	&& rm -rf /*.patch \
-	&& apk del .build-deps
+		openssl
+
+ENV RUNTIME_GO_VERSION ${RUNTIME_GO_VERSION}
```
Comment (mcuadros) [inline etc/build/etc/build/Dockerfile.build.alpine.tpl]: this code belongs to another PR

Comment (campoy) [reply etc/build/etc/build/Dockerfile.build.alpine.tpl]: ouch, E_TOO_MANY_BRANCHES
I'll fix it

**Comment by campoy** [comment]
Test added and it made me consider what to do with nil maps, which I think should also be ignored.

File: uast/node.go (hunk 1 above)
Comment (campoy) [reply uast/node.go]: done
